strapi/strapi
# strapi-helper-plugin.request do not run parseJSON for 204 No content response (#10562)

### What does it do?

204 No content response has application/json type but should not contain any body and empty body will not be qualified as valid JSON, therefore .json() should not be called for response with 204 status code.

### Why is it needed?

Simple adjustment.

### How to test it?

1. Use import { request } from 'strapi-helper-plugin';
2. Call API that returns 204 response with request
3. Should return response (before PR it would throw an error)

### Related issue(s)/PR(s)

#10562
